src/client/client.cpp

```diff
@@ -1768,7 +1768,14 @@ void Client::afterContentReceived()
 	tu_args.last_percent = 0;
 	tu_args.text_base =  wgettext("Initializing nodes");
 	tu_args.tsrc = m_tsrc;
-	m_nodedef->updateTextures(this, texture_update_progress, &tu_args);
+
+	if (!m_nodedef->updateTextures(this, texture_update_progress, &tu_args)) {
+		error_message = "Failed to update textures! Your shader code is probably wrong. Please check the log for details.";
```

Comment:
users are not the people who write the shaders
I suggest something like `"Failed to update textures! Shader compiling probably failed, check the log for details."`

Reply:
Updated